integration-tests/schemas/createSchema.test.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiRx, WsProvider } from "@polkadot/api";
+import { Keyring } from "@polkadot/api";
+
+import assert from "assert";
+
+import { AVRO_GRAPH_CHANGE } from "./scaffolding/schemaTypes";
+import { AVRO } from "./scaffolding/modelTypes";
+import { ON_CHAIN } from "./scaffolding/payloadLocation";
+import { filter, firstValueFrom, mergeMap, Observable } from "rxjs";
+import { groupEventsByKey } from "./scaffolding/helpers";
+
+describe("#createSchema", () => {
+    let apiObservable: Observable<ApiRx>;
+    let keys: any;
+
+    beforeEach(() => {
```

Comment:
I'm having issues with running the tests together. I wonder if it is because we are using two separate instances of the API. We might need to have a singleton so that we are re-using our api.

Reply:
what issues are you running into?